### DISCUSSION TITLE
Automatically re-credit the canceled reservations

### DISCUSSION THREAD

**Discussion by marclura** [sleede/fab-manager#462] [Q&A] [answered]
Is there a way to automatically refund the credits to the wallet when a reservation is canceled without passing by the refund invoice? If yes, where can I find that setting option? If not, are there hints on implementing that change in the Fabmanager? Thanks

**Comment by gnepud** [comment, marked as answer]
Hi @marclura 
We currently do not have an option to automatically refund credits to the wallet. However, I believe this is a good idea and we will discuss implementing this feature within our team. It is possible that we may develop this feature in the future.

Thanks

**Reply by marclura** [reply]
Hi @gnepud,
thanks for the answer. That would be very handy. We are managing many users in our lab, and because we are introducing the payment system soon (until now it was free), the automatic refund would be handy.
